packages/squiggle-lang/src/dist/SymbolicDist.ts

```diff
@@ -361,8 +367,14 @@ export class Cauchy extends SymbolicDist {
   sample() {
     return jstat.cauchy.sample(this.local, this.scale);
   }
-  mean(): never {
-    throw new Error("Cauchy distributions may have no mean value.");
+  mean() {
+    return NaN; // Cauchy distributions may have no mean value.
```

Comment:
Note: previous version caused playground crash.
(`project.run` intercepts the exceptions, but React components sometimes don't, and reporting the dist statistics happens through calling this method directly)